etc/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -167,6 +167,22 @@
     regexp: '^(Subsystem.*sftp.*)'
     replace: '#\1'
 
+#
+# Remove the last-login message when ssh-ing.
+#
+- replace:
+    dest: /etc/ssh/sshd_config
+    regexp: '#PrintLastLog yes'
+    replace: 'PrintLastLog no'
+
+#
+# Remove the ubuntu motd banner and related dross.
```

Comment:
Similar to above, rather than simply repeating what the code already states, it'd be more useful to explain why we want to do this configuration/removal.